### DISCUSSION TITLE
recipes/sides/cheese-souffl%C3%A9/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#512] [Comments]
# recipes/sides/cheese-souffl%C3%A9/

A collection of my recipes.

https://nicholaswilde.io/recipes/sides/cheese-souffl%C3%A9/

**Comment by nicholaswilde** [comment]
Came out a bit watery, despite baking for an extra 10 minutes. Did not rise, either. Made in a single 7" dish so perhaps using smaller dishes may help. Grated "stringy" Parmesan was used to coat the dish instead of "breadcrumb" like Parm and batter might not have been able to grab onto it since the cheese was more sparse. May need to use breadcrumbs instead. Meringue also had large bubbles and collapsed easier. Also, there was a bit of broken yolk in the whites.

**Comment by nicholaswilde** [comment]
Add link to onion pique
